### ISSUE TITLE
SoapClient in the __construct()?

### ISSUE THREAD

**Issue by morenDev** [mpociot/vat-calculator#41]
Is that really necessary to init SoapClient in the constructor?
I mean, if I need vat calculator for other reasons (check vat rate) do i really have to load 'wsdl' all the time? What about to use setSoapClient() for that reason?

**Comment by s2pidaoshin** [comment]
It would be good to have a configuration for that, and set to false if you do not want to load soap. The reason is that now http://ec.europa.eu/taxation_customs breaks, and request is taking to long. Also in future http://ec.europa.eu/taxation_customs can be changed which will always timeout the request.

**Comment by morenDev** [comment]
I'll send a pull request asap

**Comment by mpociot** [comment]
A pull request for this would be great!

**Comment by webdevvie** [comment]
I'm just using it for the vat rates. I ran into this problem too.
I decided to fix it. Only not through configuration. 

I moved the initialisation of the soap client to a separate function, then call that function when needed. If it has already been initialised or is false then it won't re-initialise it.

I think this is a more elegant solution in case you are using it as a service. 
All tests are green.
